source/shared/product/_sidebar.html.erb

```diff
@@ -1,7 +1,10 @@
 <nav class="sub-navigation">
   <ul>
     <li class="sub-navigation__item">
-      <a href="/what-is-govwifi">What is GovWifi?</a>
+      <%= link_to "What is GovWifi?", "/what-is-govwifi" %>
+    </li>
+    <li class="sub-navigation__item">
+      <%= link_to "Who GovWifi is for", "/who-govwifi-is-for" %>
```

Comment:
Perhaps this link should be `/about-govwifi/who-govwifi-is-for`?

Reply:
You're right. It looks like we've made this mistake elsewhere too. I've fixed up the `What is GovWifi?"` /  `"/about-govwifi"` page too.